website/.env

```
@@ -3,6 +3,7 @@ PUBLIC_WEBSITE_URL="https://formisch.dev"
 PUBLIC_GITHUB_URL="https://github.com/open-circle/formisch"
 PUBLIC_GITHUB_SPONSORS_URL="https://github.com/sponsors/fabian-hiller"
 
+PUBLIC_STACKBLITZ_ANGULAR_URL="https://stackblitz.com/edit/formisch-playground-angular"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)